packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/advices/template.html

```diff
@@ -1,5 +1,9 @@
 <ovh-up-sell-cross-sell
-    description="{{ ::'server_advices_dedicated_description' | translate }}"
+    data-ng-if="!$ctrl.loading"
+    description="{{ ::$ctrl.isBustCapacityLimitReached ?
+        'server_advices_limit_reached_description' :
+        'server_advices_constant_trafic_description' | translate
```

Comment:
**question**
The whole ternary condition should be in parenthesis, to be sure to apply `| translate` to the generated key no ?